src/main/java/core/basesyntax/strategy/impl/TransactionStrategyBalance.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Dao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.SaveStrategy;
+
+public class TransactionStrategyBalance implements
+        SaveStrategy<FruitTransaction, Dao<String, Integer>> {
+    @Override
+    public void save(FruitTransaction value, Dao<String, Integer> storage) {
```

Comment:
maybe it makes sense to create a Dao class field and initialize it through the constructor

Reply:
Also this will increase code coupling, so Id rather pass storage as parameter during strategy usage.

Reply:
you can create an object of the class you need(Dao) before initialization. And I do not agree with the expense of increasing ties. You will only pass the DAO object once on creation, but in your code it is passed every time

Reply:
Moved storage to constant and initialized before STRATEGY_FOR_OPERATION Map